cmd/common.go

```diff
@@ -180,9 +180,18 @@ func prettyPrint(r *check.Controls, summary check.Summary) {
 			colors[check.WARN].Printf("== Remediations ==\n")
 			for _, g := range r.Groups {
 				for _, c := range g.Checks {
-					if c.State == check.FAIL || c.State == check.WARN {
+					if c.State == check.FAIL {
 						fmt.Printf("%s %s\n", c.ID, c.Remediation)
 					}
+					if c.State == check.WARN {
+						// Print the error if test failed dut to problem with the audit command
+						// Else tests failed due to unwanted output and just print remediation
```

Comment:
It's not really "unwanted output", it's just that the audit test didn't meet the criteria. I would just remove this comment line, it's pretty clear from the code itself